storage/txcache/config.go

```diff
@@ -1,32 +1,37 @@
 package txcache
 
-import "fmt"
+import (
+	"encoding/json"
+	"fmt"
 
-// CacheConfig holds cache configuration
-type CacheConfig struct {
-	Name                       string
-	NumChunksHint              uint32
-	EvictionEnabled            bool
-	NumBytesThreshold          uint32
-	NumBytesPerSenderThreshold uint32
-	CountThreshold             uint32
-	CountPerSenderThreshold    uint32
-	NumSendersToEvictInOneStep uint32
-	MinGasPriceNanoErd         uint32
+	"github.com/ElrondNetwork/elrond-go/core"
+)
+
+// ConfigSourceMe holds cache configuration
+type ConfigSourceMe struct {
+	Name                          string
+	NumChunks                     uint32
+	EvictionEnabled               bool
+	NumBytesThreshold             uint32
+	NumBytesPerSenderThreshold    uint32
+	CountThreshold                uint32
+	CountPerSenderThreshold       uint32
+	NumSendersToPreemptivelyEvict uint32
+	MinGasPriceNanoErd            uint32
 }
 
 type senderConstraints struct {
 	maxNumTxs   uint32
 	maxNumBytes uint32
 }
 
-// TODO: perhaps add better constraints for "CountThreshold" and "NumBytesThreshold"?
-func (config *CacheConfig) verify() error {
+// TODO: Upon further analysis and brainstorming, add some sensible minimum accepted values for the appropriate fields.
+func (config *ConfigSourceMe) verify() error {
 	if len(config.Name) == 0 {
 		return fmt.Errorf("%w: config.Name is invalid", errInvalidCacheConfig)
 	}
-	if config.NumChunksHint == 0 {
-		return fmt.Errorf("%w: config.NumChunksHint is invalid", errInvalidCacheConfig)
+	if config.NumChunks == 0 {
+		return fmt.Errorf("%w: config.NumChunks is invalid", errInvalidCacheConfig)
 	}
 	if config.NumBytesPerSenderThreshold == 0 {
```

Comment:
config checks should be againts a set minimum constant.

Reply:
Fixed (part 2), but in the future we have to discuss their values, if they need to be adjusted and so on.